src/lib/viewers/archive/ArchiveExplorer.js

```diff
@@ -190,7 +190,19 @@ class ArchiveExplorer extends React.Component {
             return 0;
         });
 
-        return sortDirection === SortDirection.DESC ? sortedItems : sortedItems.reverse();
+        if (sortDirection === SortDirection.DESC) {
```

Comment:
I know this is to keep consistent with Webapp, but I've always found this sorting weird because when I sort by name I want the files/folders sorted by name, not by type and then name. Code looks good but just calling out if this is the UX we really want and design wants.